Userland/Utilities/run-tests.cpp

```diff
@@ -255,6 +256,11 @@ FileResult TestRunner::run_test_file(const String& test_path)
 
 int main(int argc, char** argv)
 {
+    // Configure each sub test runner.
+    //
+    setenv("TESTS_ONLY", "1", true);
```

Comment:
Making this change like this makes it difficult to verify that all the benchmarks still pass locally.

Could we hide this one behind a true by default flag?

Reply:
Do we really care about benchmarks? Benchmarks don't normally have any validation that can fail, so I'm not sure I see the value.

Reply:
They've broken in the past, e.g. when Andreas changed the way the Font Database works to require a WindowServer connection to be populated. I suppose when someone cares about a particular benchmark they can go about fixing any breakages, if we wanna take that stance. Automatic running of all of them would be manual anyway

Reply:
Ak, I added an option to run benchmarks.